### DISCUSSION TITLE
Free Space

### DISCUSSION THREAD

**Discussion by Teloriun** [openscopeproject/TrguiNG#62] [Q&A]
Hi

I currently use TrguiNG on my Windows 10 desktop PC to connect to daemons on a Windows 11 NUC and Synology NAS ( slowly migrating over so running concurrently atm).

NAS 'tab' is correctly showing the remaining hard drive space, but the NUC is not.

I'm guessing this is a Windows 11 issue but wondered if anyone has encountered this before or any ideas for fixing? 

NAS: 
![image](https://github.com/openscopeproject/TrguiNG/assets/13600266/8d043791-b118-4bf5-be81-e7720f3c0e44)
NUC:
![image](https://github.com/openscopeproject/TrguiNG/assets/13600266/19a3d881-2c9e-4aa6-98f0-7029642b7fb3)

edit:
NAS is running Transmission v3.00 / NUC v4.0.3

**Comment by qu1ck** [comment]
Do you have a default download directory set in your NUC instance of transmission? The way transmission checks for free space is by checking free space of a drive that has the default download dir. If that dir does not exist or is not on a local drive it may not be able to calculate free space.

**Comment by Teloriun** [comment]
Hi qu1ck.  Yes, the location is present.

on NUC:
![image](https://github.com/openscopeproject/TrguiNG/assets/13600266/f6bad7b5-c5a7-421c-a338-a5868e0b2501)

on TrguiNG:
![image](https://github.com/openscopeproject/TrguiNG/assets/13600266/637637f0-4c02-41ef-9c65-99b09c59b8fe)

**Reply by qu1ck** [reply]
And that folder is normal directory, not a mapped network location?

I just tested on windows 10 with daemon version 4.0.3 and I get correct free space corresponding to the drive of default download location.
I have no win 11 to test this but I'm skeptical that it would be the issue.

**Comment by Teloriun** [comment]
<edit>
I think I've mixed things up - the TGui is using the mapped drive on my local desktop to connect to the NUC.  I was thinking that it was reading the daemon settings, but it's not is it.  

So yes, it's a mapping thing.. as my PC is only connecting to the specific downloads folder so presumably won't be able to see the rest of the c drive and its space.

Apologies for wasting your time. :(

**Reply by qu1ck** [reply]
TrguiNG does not calculate free space itself, it just shows what transmission calculates. Transmission will calculate it from the perspective of the machine where the daemon is running, obviously, so if the download directory is local to that machine it should have no trouble calculating free space and TrguiNG will show that.

**Comment by Teloriun** [comment]
Fresh day, fresh head.  Started from scratch and reinstalled/setup everything (apart from sharing the drive) and it's working.  I'll push the files from the NUC rather than pull - I clearly broke something yesterday somehow. :/
